src/objects.c

```diff
@@ -1328,8 +1328,13 @@ Obj SET_TYPE_COMOBJ_Handler (
         RetypeBag( obj, T_ACOMOBJ );
         CHANGED_BAG( obj );
         break;
+      default:
+	ErrorMayQuit("You can't make component object from a %s.",(Int)TNAM_OBJ(obj),0L);
```

Comment:
Formatting: could you please avoid using a tab there, and indented using spaces? Also, spaces after the commas